vars/pythonSetupPyPipeline.groovy

```diff
@@ -215,6 +215,26 @@ def call(Map pipelineParams) {
                                 pipelineParams.dockerRepo, moduleName, moduleVersion)
             }
           } // Deploy Docker
+
+          // Deploy the html documents to the docs branch, which when using "BitBucket Pages" allows serving the documentation directly
+          stage("Deploy Docs") {
+            when {
+              beforeAgent true
+              allOf {
+                branch "master"
+                expression { params.doRelease }
+              }
+            }
+            steps {
+              sshagent([pipelineParams.sshAgentUser]) {
+                script {
+                  sh "ghp-import -m \"Documentation update to $moduleVersion\" -p -b docs build/sphinx/html"
```

Comment:
You will also be adding this to https://github.com/tomtom-international/cookiecutter-python/blob/master/%7B%7Bcookiecutter.project_name%7D%7D/requirements_dev.txt I suppose, so that it will become part of the requirements for development?

Reply:
yes, I need to add **ghp-import** to the https://github.com/tomtom-international/cookiecutter-python/blob/master/%7B%7Bcookiecutter.project_name%7D%7D/requirements_dev.txt.
Also to other repos generated from **cookiecutter-python** template.